charts/Geomap.stories.js

```diff
@@ -16,16 +16,111 @@ GettingStarted.args = {
   data: []
 };
 
+export const ChangingCoordinatePointSizes = Template.bind({});
+ChangingCoordinatePointSizes.args = {
+  data: "https://d3plus.org/data/city_coords.json",
+    groupBy: "slug",
+    colorScale: "dma_code",
+    label: funcify(function(d) {
+        return d.city + ", " + d.region;
+      }, `function(d) {return d.city + ", " + d.region}`),
+    point: funcify(function(d) {
+      return [d.longitude, d.latitude];
+    }, "function(d) {return [d.longitude, d.latitude]}"),
+    pointSize: funcify(function(d) {
+      return d.dma_code;
+    }, "function(d) {return d.dma_code}"),
+    pointSizeMin: 1,
+    pointSizeMax: 10
```

Comment:
Indentation looks too big here?

Reply:
Ooops! I moved the whole object a tab right 😞  Good catch!